webapp/templates/base.html.twig

```diff
@@ -8,6 +8,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 
     <link rel="stylesheet" href="{{ asset("css/blog.css") }}">
+    <link rel="stylesheet" href="{{ asset("css/help.css") }}">
```

Comment:
toto je potreba pouze na help strance ne? -> tady se to bude includovat vsude

Reply:
castokrat se to dela tak, ze zde v parent zadefinuje sekci, ktera bude prazdna (treba extra_css) a tuto sekci potom v child sablone nadefinujes.

Snad to twig umi - https://stackoverflow.com/questions/32585897/changing-css-selector-for-parent-template-using-twig